src/pages/public_registration/PublicRegistrationContent.tsx

```diff
@@ -118,9 +117,9 @@ const PublicRegistrationContent: FC<PublicRegistrationContentProps> = ({ registr
   const selectedLicense = licenses.find((license) => license.identifier === currentLicense?.identifier);
 
   return (
-    <ContentPage>
+    <div>
       <PublicRegistrationStatusBar registration={registration} refetchRegistration={refetchRegistration} />
-      <Heading>{mainTitle}</Heading>
+      <RegistrationPageHeader>{mainTitle || `[${t('common:missing_title')}]`}</RegistrationPageHeader>
```

Comment:
```suggestion
      <RegistrationPageHeader>{mainTitle ?? `[${t('common:missing_title')}]`}</RegistrationPageHeader>
```

Reply:
prøvde det først, men det virket ikke. `??` virker sikkert bare for `null`, mens her er det tom streng